locals.tf

```diff
@@ -74,7 +73,12 @@ locals {
       disk_size              = var.gpuNodeDiskSize
       custom_ami_id          = data.aws_ami.al2gpu_ami.image_id
       create_launch_template = true
-      post_userdata          = local.gpuPostUserData
+      block_device_mappings = [{
+        device_name           = "/dev/sda1"
+        volume_type           = "gp2"
```

Comment:
I know this is not topic of PR but did you thought about switching to gp3 storage instead of gp2?
Link to AWS doc: https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-storage-compare-volume-types.html

Reply:
changed it to gp3